toolkit/tools/internal/rpm/rpm.go

```diff
@@ -536,7 +536,7 @@ func buildAllSpecsList(baseDir string) (specPaths []string, err error) {
 
 	specPaths, err = filepath.Glob(specFilesGlob)
 	if err != nil {
-		logger.Log.Errorf("Failed while trying to enumerate all spec files with (%s). Error: %v.", specFilesGlob, err)
+		return nil, fmt.Errorf("Failed while trying to enumerate all spec files with (%s). Error: %w", specFilesGlob, err)
```

Comment:
For all places where we switched to `%w` for printing nested errors, could you do `\n%w` instead? We've tried to follow that in other places and it makes it easier to read the nested errors. Especially, if there's multiple levels of nesting.

Reply:
My comment as above, should we leverage the native Wrapping of errors instead of string manipulations ?